src/main/java/seedu/manager/ui/Ui.java

```diff
@@ -34,16 +30,16 @@ public void greetUser() {
     /**
      * Gets input from the user, and executes commands based on that input
      */
-    public void getCommands() {
-        boolean isGettingCommands = true;
-
-        while (isGettingCommands) {
-            System.out.print(COMMAND_PROMPT_MESSAGE);
-            String rawInput = this.userInput.nextLine();
-            Command command = parser.parseCommand(rawInput);
-            CommandOutput output = command.execute();
-            output.printMessage();
-            isGettingCommands = !output.getCanExit();
-        }
+    public String getCommands() {
+        System.out.print(COMMAND_PROMPT_MESSAGE);
+        return this.userInput.nextLine();
+    }
+
+    /**
+     * show the output of command to the users.
+     */
+    public void showOutputToUser(CommandOutput output){
+        System.out.println(output.getMessage());
+        System.out.println();
```

Comment:
Could this additional call to println() be changed to an additional System.lineSeparator in the previous call?

Reply:
Thanks for advice. 
A new commission is done.